AnnService/src/Helper/AsyncFileReader.cpp

```diff
@@ -6,6 +6,49 @@
 namespace SPTAG {
     namespace Helper {
 #ifndef _MSC_VER
+        void SetThreadAffinity(int threadID, std::thread& thread, char socketStrategy, char idStrategy)
+        {
+#ifdef NUMA
+            int numGroups = numa_num_task_nodes();
+            int numCpus = numa_num_task_cpus() / numGroups;
+
+            int group = threadID / numCpus;
+            int cpuid = threadID % numCpus;
+            if (socketStrategy == 1) {
```

Comment:
can we name these strategies by descriptive enum rather than int value?

Reply:
sure